gnome-initial-setup/pages/language/gis-language-page.c

```diff
@@ -42,8 +42,6 @@
 
 struct _GisLanguagePagePrivate
 {
-  GtkWidget *logo;
```

Comment:
This removes the welcome message and the gnome logo from the language page, which is the first page upstream. But it is not true on the endless fork; the first page is `branding-welcome`. Maybe we could squash this with the commit that adds the `branding-welcome` page. I left these commit separated as a precaution.